src/trace/cltrace.cpp

```diff
@@ -807,6 +1366,57 @@ cl_int clFinish(cl_command_queue queue) {
     return ret;
 }
 
+void* clSVMAlloc(cl_context context, cl_svm_mem_flags flags, size_t size,
+                 unsigned int alignment) {
+    auto ret = PFN_clSVMAlloc(context, flags, size, alignment);
+
+    Call call(oclapi::command::SVMALLOC);
+    call.record_object_use(context);
+    call.record_value(flags);
+    call.record_value(size);
+    call.record_value(alignment);
+    call.record_return_map_pointer_creation(ret);
```

Comment:
```suggestion
    call.record_return_map_pointer_creation(ret); // FIXME
```

This will need special handling.